about/aboutMobile.css

```diff
@@ -0,0 +1,482 @@
+@import url('https://fonts.googleapis.com/css2?family=Archivo+Black&display=swap');
+@import url('http://fonts.cdnfonts.com/css/coco-gothic');
+@import url('https://fonts.googleapis.com/css2?family=Archivo+Black&family=Lato:wght@100;300;400;900&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
```

Comment:
- [ ] It would be nice to use a media query for your project. The recommended media query for this project is 
`@media screen and (min-width: 768px)`. Kindly implement it as there are no media queries in your project. It is good practice to design for mobile first then extend to **_desktop_** version and not the other way around.

- [ ] I also noticed that the app is not responsive `@768` and texts overlap at the mobile view. Kindly make it responsive and fix the case of the texts overlapping. 

### View @768px

![image](https://user-images.githubusercontent.com/77106962/153725979-76b4ddaa-b45b-4883-b4c5-be0953e01c73.png)

### Mobile View (Home Page)

![image](https://user-images.githubusercontent.com/77106962/153726002-93ec7b65-98d1-480f-8a8d-9ac72059ce58.png)

### Mobile View (About Page)

![image](https://user-images.githubusercontent.com/77106962/153726055-671d3c32-2b48-41d2-a316-75616e1373be.png)